packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -25,13 +25,23 @@ const Overview = decorate([
     legacyJobs: subscribeJobs,
   }),
   provideState({
+    initialState: () => ({
+      scrollIntoJobs: undefined,
```

Comment:
There might be a chance that it's still `undefined` when it's called.

Reply:
A fetch for logs takes more time than the ref fetch, and, it's a human action so this case will not be probable.